### PR TITLE
lesson-4-4

### PR DESCRIPTION
**Pull request by Manizha-khorram**

### CODE REVIEW

File: Style.css [hunk 1]
```diff
@@ -1,12 +1,156 @@
...
+
+}
+
+footer {
+  background-image: url("f.jpg");
+  padding: 20px;
+}
+
+/* Although it was not part of assignment.I couldnt understand how to select the element which was created by js please giud me!
+  form li:nth-of-child(2){
```
Comment (p34c310v3) [inline Style.css:137]: Hmm, from what I'm seeing, the messages (created by the form) aren't children of the `<form>`, looks like they're under `<section id="messages">`, so I think something like: `#messages li` would work for a selector. :thinking: :smiley:

File: index.html [hunk 2]
```diff
@@ -1,62 +1,68 @@
...
+    </section>
+
+    <section class="myInfo">
+        <h2><a href="#Experience">Experience</a></h2>
+        <p>I have worked as a system development analyst at NPA (National Procurement Authority) and USWDP (University
+            Work Force Development Program) in Afghanistan.</p>
+    </section>
+
+    <section id="skills" class="myInfo">
+        <h2><a href="#Connect">Skills</a></h2>
```
Comment (p34c310v3) [inline index.html:40]: I see the section below is "#Connect", I wonder if this should be "#Skills" ? :thinking: :smile:

**Comment by p34c310v3** [comment]
Btw, I'm using Firefox and I'm seeing that your name is being clipped at a larger screen size. :smile: 

khorram

**Review by p34c310v3 — APPROVED** [review]
I pulled this down and looked at it locally and I really like your style changes! :raised_hands: :tada: :confetti_ball:

**Comment by Manizha-khorram** [comment]
> I pulled this down and looked at it locally and I really like your style changes! 🙌 🎉 🎊

Thanks alot for all your comments

**Pull request merged**